packages/components/src/layouts/Navigation/Tab/TabBar/DesktopTabItem.tsx

```
@@ -90,102 +90,99 @@
+            p="$0.5"
+            m={-3}
+            testID="browser-bar-options"
+            onPress={onClose}
+          />
+        ) : null}
+        {actionList ? (
+          <ActionList
+            title=""
+            placement="right-start"
+            sections={actionList}
+            renderTrigger={<></>}
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)